wp-content/themes/pub/wporg-learn-2020/css/components/_page-lesson-plan.scss

```diff
@@ -0,0 +1,137 @@
+.lesson-plan {
+	&-taxonomy {
+		&-header {
+			width: 100%;
+			flex-basis: 100%;
+			font-weight: 400;
+			color: #555D66;
+			font-size: 1.375rem;
+			line-height: 1.36;
+			margin-bottom: 0.75rem;
+		}
+
+		&-description {
+			margin-bottom: 2rem;
+			width: 100%;
+			flex-basis: 100%;
+			color: #555D66;
+		}
+
+		.card-grid  {
+			margin: 2rem -0.5rem;
+			justify-content: flex-start;
+
+			.card {
+				text-align: center;
+				margin-top: 0;
+				margin-bottom: 1rem;
+
+				a:hover,
+				a:focus {
+					text-decoration: none;
+				}
+
+				.dashicons {
+					width: 80px;
+					height: 80px;
+					font-size: 80px;
+					margin: 0 auto 32px;
+					color: #606A73;
+				}
+
+				.taxonomy-title {
+					margin-bottom: 0;
+					font-weight: 700;
+
+					a {
+						color: #555D66;
+						text-decoration: none;
+					}
+				}
+			}
+		}
+
+		&.categories {
+			.card-grid  {
+				margin-top: 3rem;
+			}
+		}
+
+		&.last-grid {
+			.card-grid {
+				margin-bottom: 3rem;
+				margin-top: 3.5rem;
+			}
+		}
+	}
+
+	&-two-col {
+		@media only screen and (min-width: $breakpoint-tablet) {
+			margin-left: -0.5rem;
+			margin-right: -0.5rem;
+
+			> * {
+				margin: 0.5rem;
+				width: calc(50% - 1rem);
+				flex-basis: calc(50% - 1rem);
+			}
+		}
+
+		.list {
+			list-style-type: none;
+			margin-left: 0;
+			margin-top: 2rem;
+			margin-bottom: 0;
+
+			li {
+				padding: 1.0625em;
+				margin-bottom: 0.5rem;
+				background: #FBFBFC;
+				border: 1px solid #CCD0D4;
+				border-radius: 3px;
+				position: relative;
+
+				&:last-child {
+					margin-bottom: 0;
+				}
+
+				a {
+					text-decoration: none;
+					font-weight: 700;
+					color: #555D66;
+					display: flex;
+					line-height: 1.5rem;
+				}
+
+				.dashicons {
+					vertical-align: middle;
+					font-size: 2rem;
+					width: 1.875rem;
+					height: 1.875rem;
+					position: absolute;
```

Comment:
Absolute positioning is brittle with longer names. The anchor wrapping this already has `display:flex` so we should be able to use that to position this to the right. We may need to wrap the name in a span with `flex: 1`